pkg/events/derive/symbols_collision.go

```diff
@@ -29,8 +29,10 @@ func SymbolsCollision(soLoader sharedobjs.DynamicSymbolsLoader, policies *policy
 	symbolsCollisionFilters := map[string]filters.Filter{}
 
 	// pick white and black lists from the filters (TODO: change this)
-	for _, policies := range policies.Map() {
-		f := policies.ArgFilter.GetEventFilters(events.SymbolsCollision)
+	it := policies.CreateAllIterator()
+	for it.HasNext() {
```

Comment:
Suggestion for a different and maybe more readable way to write it (it is more C like, so I am not sure if it is really better):
```Go
for it := policies.CreateAllIterator; it.HasNext(); p := it.GetNext() {
	f := p.ArgFilter.GetEventFilters(events.SymbolsCollision)
	maps.Copy(symbolsCollisionFilters, f)
}
```
I think it has the pro of not exposing the `it` variable after the loop, as it is only created for iterating over it.